/bb_tracker/bb_tracker:
   ros__parameters:
      fps: 30
      track_buffer: 30
      track_thresh: 0.5
      high_thresh: 0.6
      match_thresh: 0.8
      fixed_frame: sensors_home